src/main/java/com/godaddy/sonar/ruby/simplecovrcov/data/CoverageReport.java

```diff
@@ -0,0 +1,27 @@
+package com.godaddy.sonar.ruby.simplecovrcov.data;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Created by sergio on 2/8/17.
+ */
+public class CoverageReport {
+    private Collection<Reporter> reporters = new ArrayList<>();
+
+    public void addReporter(Reporter reporter) { this.reporters.add(reporter); }
+
+    public Collection<Reporter> getReporters() {
+        return reporters;
+    }
+
+    public Boolean hasSeveralReporters() { return this.reporters.size() > 1; }
+
+    public Reporter getFirstReporter() {
```

Comment:
You are right 👍  This method is not needed anymore, as like as `hasSeveralReporters`. I'll eliminate them.